osutil/squashfs/fstype.go

```diff
@@ -0,0 +1,82 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package squashfs
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/snapcore/snapd/osutil"
+)
+
+// useFuse detects if we should be using squashfuse instead
+var useFuse = realUseFuse
+
+func realUseFuse() bool {
```

Comment:
typically we call these: useFuseImpl

Reply:
Yeah, I inherited this. But I fixed it now.